src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -175,18 +175,23 @@ public void onActivityCreated(Activity activity, Bundle bundle) {
 
         @Override
         public void onActivityStarted(Activity activity) {
+            ConnectivityIndicator.start(mContext);
+            mCurrentActivity = activity;
```

Comment:
mEditState is a HashSet and therefore is unordered. I need to keep track of most recent active Activity